parse.go

```diff
@@ -0,0 +1,115 @@
+package main
+
+import (
+	"encoding/csv"
+	"fmt"
+	"io"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/cockroachdb/apd/v2"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+type Record struct {
+	Address                 sdk.AccAddress
+	TotalAmount             apd.Decimal
+	StartTime               time.Time
+	NumMonthlyDistributions int
+}
+
+const (
+	SecondsPerYear  = 31556952
```

Comment:
Shouldn't this be 31536000? (60*60*24*365)

Or 31557600 if we want to account for leap year? (`(60*60*24*365)*(3/4)+(60*60*24*366)*(1/4)`)

Reply:
https://www.rapidtables.com/calc/time/seconds-in-year.html

Using 365.2425 days/year